docs/src/adoc/index.adoc

```diff
@@ -7694,6 +7694,9 @@ link:{jdkdocs}/java.base/java/time/Instant.html[Instant^] or link:{jdkdocs}/java
 It also configures SQL array type support for `int`, `long`, `float`, `double`,
 `String`, and `UUID`.
 
+If link:https://github.com/pgvector/pgvector-java[PGvector-java^] is available, Jdbi will register bindings
+and mappers for `PGvector`, `PGhalfvec`, and `PGsparsevec`.
```

Comment:
why not PGbit? 

Consider linking https://javadoc.io/doc/com.pgvector/pgvector/latest/com/pgvector/ pages for javadocs.

Reply:
Because it's not mentioned anywhere in the pgvector readme, so I didn't realize it even exists.